config/rbac/role.yaml

```diff
@@ -132,18 +132,11 @@ rules:
   - flows.netobserv.io
   resources:
   - flowcollectors/status
+  - flowmetrics/status
   verbs:
   - get
   - patch
   - update
```

Comment:
u dropped `watch` verb that was used with fm is that no longer needed ?

Reply:
no `flowmetrics` still has `watch` look at line 124, it's now having same perms as `flowcollectors`